Stroking/Stroke.js

```diff
@@ -100,7 +100,7 @@ function sendNewStrokeInstruction() {
             sendMessage('Only use one finger for now and rub it up and down your %Cock% %Grin%');
             break;
         case 6:
-            let lubeType = getAssLubeType(getMood(), 30);
+             lubeType = getAssLubeType(getMood(), 30);
```

Comment:
Why?

Reply:
it was crashing when the file loaded (not observable but in the startup logs)  it may already be declared.